cypress/e2e/prices/eco_atm_prices.cy.js

```diff
@@ -0,0 +1,208 @@
+import { getBaseUrl } from "../../support/premium_price_utilities";
+
+const env = Cypress.env("ENV");
+const omni_api_base = getBaseUrl(env, "omni");
+const graphqlApi = `${omni_api_base}/graphql`;
```

Comment:
remove all the constantns except the graphqlApi and hard code the value , PROD URL  .  we can remove the utilities files as well